std_msgs/Header header

float32 azimuth
float32 elevation
# TODO: covariance